src/dotnet/APIView/APIViewWeb/Client/src/shared/signalr.ts

```diff
@@ -32,6 +38,23 @@ $(() => {
     hp.addToastNotification(notification);
   });
 
+  // receiver/client side of comment refresh 
+  connection.on("ReceiveComment", (reviewId, elementId, partialViewResult) => {
+    let result = hp.getReviewAndRevisionIdFromUrl();
+    let currReviewId = result["reviewId"];
+
+    if (currReviewId != reviewId) {
+      return;
+    }
+
+    var rowSectionClasses = hp.getCodeRowSectionClasses(elementId);
+    hp.showCommentBox(elementId, rowSectionClasses, undefined, false);
```

Comment:
You'll need one more check here for swagger reviews.  If the elementId is not found on the page, or if it also has `d-none ` class then do nothing with the received comment.

Reply:
By doing nothing, do you mean no autorefresh?